analysis/jira__issue_transition_cumulative_flow_analysis.sql

```diff
@@ -0,0 +1,61 @@
+with field_history as ( 
+
+  select * 
+  from {{ ref('jira__timestamp_issue_field_history') }}
+
+),
+
+status_transitions as (
+
+  select *
+  from {{ ref('jira__issue_status_transitions') }}
+),
+
+---if you prefer to keep analytics at the status level, substitute 'status' for 'status_category' throughout this model
+joined as (
+
+  select 
+    transitions.transition_at,
+    transitions.issue_id,
+    transitions.status_category_name,
+    -- edit field_history columns as desired based on the issue_field_history_columns var for your dbt project
+    -- project and team have been chosen for this example
+    field_history.team,
+    field_history.project
+  from status_transitions as transitions
+    join field_history 
+      on transitions.issue_id = field_history.issue_id
+      and transitions.transition_at = field_history.valid_from
```

Comment:
small indentation fix

```suggestion
  from status_transitions as transitions
  join field_history 
      on transitions.issue_id = field_history.issue_id
      and transitions.transition_at = field_history.valid_from
```